system-test/spanner.ts

```diff
@@ -29,7 +29,7 @@ const {Spanner} = require('../src');
 const PREFIX = 'gcloud-tests-';
 const RUN_ID = uuid.v1().split('-').shift();  // get a short uuid
 const LABEL = `gcloud-tests-${RUN_ID}`;
-const spanner = new Spanner({projectId: process.env.GCLOUD_PROJECT});
+const spanner = new Spanner();
```

Comment:
These are the failures removing this line causes:

```
1) Spanner
       Instances
         should have created the instance:
      + expected - actual
      -projects/long-door-651/instances/gcloud-tests-instance-28be8ad0
      +projects/{{projectId}}/instances/gcloud-tests-instance-28be8ad0
  2) Spanner
       Databases
         should have created the database:database-28bfc350'
      + expected - actual
      -projects/long-door-651/instances/gcloud-tests-instance-28be8ad0/databases/gcloud-tests-database-28bfc350
      +projects/{{projectId}}/instances/gcloud-tests-instance-28be8ad0/databases/gcloud-tests-database-28bfc350
```

These failures are because the library is correctly replacing the `{{projectId}}` token with the actual project ID. It just makes it easier to be explicit with the project ID in the constructor to avoid changing the tests to parse out the project ID from the resource ID we create.

Reply:
🤦‍♂️